lib/screenplay/screens_config.ex

```diff
@@ -24,22 +24,14 @@ defmodule Screenplay.ScreensConfig do
     Supervisor.init(children, strategy: :one_for_one)
   end
 
-  @spec screen(String.t()) :: Screen.t() | nil
-  def screen(screen_id) do
-    Cache.get(screen_id)
-  end
-
-  @spec screen_ids() :: list(String.t())
-  def screen_ids do
-    Cache.all(nil, return: :key)
-  end
-
+  @spec screens() :: list(Screen.t())
```

Comment:
@digitalcora Flagging this just in case it was missed. For some reason, GH is not flagging the comment as outdated after I pushed up the change.

Reply:
Oh, weird. I know this has happened to me before but it seems to happen _just_ infrequently enough that I've forgotten by the next time. (like once a year?)